src/Rekono/Provider.php

```diff
@@ -67,6 +67,10 @@ protected function getUserByToken($token)
      */
     protected function mapUserToObject($user)
     {
+        if (!isset($user['name'])) {
+            $user['name'] = null;
+        }
+
         return (new User())->setRaw($user)->map([
             'id'    => $user['sub'],
             'name'  => $user['name'],
```

Comment:
Why not just `'name'  => $user['name'] ?? null,` ?

Reply:
👍 Thank you!